src/bluesearch/entrypoint/database/add.py

```diff
@@ -4,6 +4,9 @@
 
 import sqlalchemy
 
+from bluesearch.database.identifiers import generate_uid
+from bluesearch.utils import load_spacy_model
+
```

Comment:
Maybe we should discuss our best practises here, I think we can consider running all import within the actual entry-point function, i.e. inside the `run`-function.

I'm mentioning this because inside `parent.py` we construct the CLI by importing all subparsers and so all imports in the module scope of the subparser module will be run, which will eventually slow down the whole CLI initialization.

Reply:
Done in 5787623.